insertion_sort/insertion_sort.c

```diff
@@ -0,0 +1,35 @@
+/*
+*  *arr --> array to be sorted
+*  arr_size -- > size of array
```

Comment:
Please remove `-->` and `-- >`. `*arr` should be replaced with `arr` since the variable is `arr`.